src/mapping/impl/BasisFunctions.hpp

```diff
@@ -148,17 +129,13 @@ class Gaussian {
     PRECICE_CHECK(math::greater(_supportRadius, 0.0),
                   "Support radius for radial-basis-function gaussian has to be larger than zero!");
 
-    _deltaY          = hasCompactSupport() ? evaluate(supportRadius) : 0;
+    if (supportRadius < std::numeric_limits<double>::infinity()) {
```

Comment:
Why using `std::numeric_limits<double>::infinity()` here but `std::numeric_limits<double>::max()` in the `NoCompactSupportBase`?

Reply:
Good point.
I could imaging that working with the maximal value of double is a numerical nightmare.
Infinity may be a better pick.

We should probably investigate this in the future.

Reply:
I agree that `std::numeric_limits<double>::infinity()` would be better.

Reply:
I'll open an issue regarding this.